humanhash.py

```diff
@@ -139,31 +140,38 @@ def compress(bytes_, target):
 
             >>> bytes_ = [96, 173, 141, 13, 135, 27, 96, 149, 128, 130, 151]
             >>> list(HumanHasher.compress(bytes_, 4))
-            [205, 128, 156, 96]
+            [64, 145, 117, 21]
 
-        Attempting to compress a smaller number of bytes to a larger number is
-        an error:
+        If there are less than the target number bytes, return input bytes
 
-            >>> HumanHasher.compress(bytes_, 15)  # doctest: +ELLIPSIS
-            Traceback (most recent call last):
-            ...
-            ValueError: Fewer input bytes than requested output
+            >>> list(HumanHasher.compress(bytes_, 15))  # doctest: +ELLIPSIS
+            [96, 173, 141, 13, 135, 27, 96, 149, 128, 130, 151]
         """
 
         bytes_list = list(bytes_)
 
         length = len(bytes_list)
-        if target > length:
-            raise ValueError("Fewer input bytes than requested output")
```

Comment:
I'm curious, why did you remove this check?

I might flip this check back in the next version.